CMakeLists.txt

```diff
@@ -466,6 +466,9 @@ if(NO_JIT_SH)
     unset(NO_JIT_SH CACHE)      # don't cache
     unset(BuildJIT CACHE)       # also clear it just in case
     add_definitions(-DDISABLE_JIT=1)
+elseif(DISABLE_JIT)
+    unset(BuildJIT CACHE)       # also clear it just in case
```

Comment:
Setting to 0 (or `OFF`, or `FALSE`) would have the same effect, and I think is more preferable when not used from the script (script attempts to unset everything it can by default).

Reply:
Updated